mne_bids/tests/test_mne_bids.py

```diff
@@ -90,6 +93,9 @@ def test_fif():
     # now force the overwrite
     write_raw_bids(raw, bids_basename2, output_path, events_data=events_fname,
                    event_id=event_id, overwrite=True)
+    raw, events, _ = read_raw_bids(bids_basename2 + '_meg.fif', output_path)
+    events2 = mne.read_events(events_fname)
```

Comment:
@agramfort any idea what event_id = 0 corresponds to in the MNE sample events file? I couldn't find that info here: https://martinos.org/mne/stable/manual/sample_dataset.html#ch-sample-data.

Reply:
It cannot be 0. I don’t follow

Reply:
Try this:

```py
import os.path as op
import mne
from mne.datasets import testing

data_path = testing.data_path()
events_fname = op.join(data_path, 'MEG', 'sample', 'sample_audvis_trunc_raw-eve.fif')
events = mne.read_events(events_fname)
print(events[:, 2])
```

I got:

```py
[ 2  0  3  0  1  0  4  0  2  0  3  0  1  0  4  0  2  0  3  0  1  0  4  0
  2  0  3  0  1  0  5  0  2  0 32  0  3  0  1  0  4  0  2  0  3  0  1  0
  4  0]
```